design/ceph/ceph-nfs-ganesha.md

```diff
@@ -129,6 +130,13 @@ spec:
             configMap:
               name: rook-ceph-nfs-organization-sssd-config
               defaultMode: 0600 # mode must be 0600
+        sssdGenericFiles:
+          volumeSource:
+            # example
+            configMap:
+              name: rook-ceph-nfs-organization-sssd-ca-bundle
+              defaultMode: 0600 # mode must be 0600
+            mountPath: /etc/local/certs
```

Comment:
I have concerns that allowing users to specify any mountPath could allow them to pretty easily mess up SSSD's ability to run. I considered a similar approach for the config files. I think it might be better to put other files into a specific directory to limit users' ability to mess things up for themselves.